index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="container">  
+  <header>
+    <div class="top-header"><img src="images/Time.png" alt="time">
+    <img src="images/wifi.png" alt="wifi">
+  </div>
+    <div class="bottom-header">
+      <h1>Welcome</h1>
```

Comment:
The logo `welcome` should be wrapped with an `a` tag instead of the `h1` tag, this is because later on, we will need to add a link that takes us to the main page of the portfolio

- [ ] We should use the correct font for the navbar as well. the correct font for this section is `Inter` font, kindly import it and use it everywhere where it applies in the Figma template.

Reply:
Alright, done! Thanks!